docs/app/pages/Components/Chips/Chips.vue

```diff
@@ -156,6 +164,15 @@ export default {
             type: 'Boolean',
             description: 'Always check if there is a duplicated chip while changing the input value, or check it only on insertion',
             defaults: 'false'
+          },
+          {
+            name: 'md-format',
+            type: 'Function',
+            description: [
+              'Formatter before chip insertion. Effects to insertion and duplicated-checking.',
+              '@param {<code>String</code>} input value'
```

Comment:
Maybe we can explicitly say that this function will receive this parameter. There is an example on md-table component

Reply:
fix ca3ac204447af756cbd22fc7c69756623f3a200b